src/pages/storage/StorageVolumes.tsx

```diff
@@ -130,40 +175,78 @@ const StorageVolumes: FC = () => {
       ],
       sortData: {
         name: volume.name,
+        pool: volume.pool,
         contentType: volume.content_type,
-        type: volume.type,
+        type: volumeType,
         createdAt: volume.created_at,
-        location: volume.location,
+        usedBy: volume.used_by?.length ?? 0,
       },
     };
   });
 
-  return (
+  const pagination = usePagination(rows);
+
+  if (isLoading) {
+    return <Loader text="Loading storage volumes..." />;
+  }
+
+  return volumes.length === 0 ? (
+    <EmptyState
+      className="empty-state"
+      image={<Icon name="mount" className="empty-state-icon" />}
+      title="No volumes found"
+    >
+      <p>Storage volumes will appear here</p>
```

Comment:
I wonder why in some empty states we have the formula "X will appear here", and in some other we have "There are no X (in/for/of Y)". :thinking: